src/wp-includes/widgets/class-wp-widget-tag-cloud.php

```diff
@@ -53,6 +53,8 @@ public function widget( $args, $instance ) {
 			}
 		}
 
+		$default_title = $title;
```

Comment:
Haha yeah, although to be fair the `$default_title` variable only gets read if a filter hooked into `widget_title` returns something empty. Otherwise it's always using the taxonomy label as the fallback.